opmon/firefox-ios-beta-health.toml

```
@@ -10,15 +10,20 @@ metrics = ['dirty_startup', 'large_file_write', 'hang_exception', 'cpu_exception
 [project.population]
 data_source = "baseline_v2"
 monitor_entire_population = true
-channel = "beta"
-dimensions = ["release"]
+dimensions = ["release", "channel"]
 
 [dimensions.release]
 data_source = "baseline_v2"
 select_expression = "client_info.app_display_version"
 friendly_name = "Release Version"
 description = "Release version"
 
+[dimensions.channel]
+data_source = "main"
```

```suggestion
data_source = "baseline_v2"
```

`main` is only for FF desktop telemetry data, but `baseline` has channel information too

---

Copy paste error, good catch 🙏